lib/venue.rb

```diff
@@ -1,2 +1,40 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+
+    @patrons = []
+  end
+
+
+  def add_patron(patron)
+    patrons << patron
+  end
+
+
+  def yell_at_patrons
+  upcase_patrons = []
+    patrons.each do |patron|
+      upcase_patrons << patron.upcase
+    end
+  upcase_patrons
+  end
+
+  def over_capacity
+    if patrons.length > capacity
```

Comment:
Nice implementation of the `if/else`. Since you are using the `>` operator it will return a boolean. So you can simply write this like:
```
patron.length > capacity
```